R/source.R

```diff
@@ -102,29 +102,41 @@ read_connection <- function(con) {
   read_connection_(con)
 }
 
-standardise_path <- function(path) {
+standardise_path <- function(path, input = TRUE) {
   if (!is.character(path))
     return(path)
 
   if (grepl("\n", path))
     return(path)
 
   if (is_url(path)) {
+    if (requireNamespace("curl", quietly = TRUE)) {
+      con <- curl::curl(path)
```

Comment:
I put the message below for the fallback, you are saying you want the message if we are using curl instead?